package.json

```diff
@@ -10,7 +10,8 @@
     "build": "webpack",
     "lint": "eslint src",
     "mocha": "NODE_PATH=./src mocha --compilers babel-core/register --require src/test/.setup.js --recursive './{,!(node_modules)/**/}*.tests.js*'",
-    "start": "npm run build && node server.js",
+    "start:dev": "webpack --watch",
+    "start:prod": "npm run build",
```

Comment:
Do we not need `&& node server.js` like in the previous start command?

Reply:
I don't think so since this serves the static html file which uses the bundled js and css.

Reply:
The `server.js` file is deleted as part of this PR.